### PR TITLE
[IE Samples] Excluded mandatory setting of layout

### PR DESCRIPTION
**Pull request by mgrdv**
Details:
 - excluded the setting of layouts from preprocessing

Tickets:
 - 78265

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
@mgrdv there are failed tests for speech sample

**Comment by vladimir-dudnik** [comment]
@mgrdv please rebase

File: samples/cpp/speech_sample/main.cpp [hunk 1]
```diff
@@ -114,17 +114,26 @@ int main(int argc, char* argv[]) {
...
+                proc.input(i).tensor().set_element_type(ov::element::f32);
             }
             for (int i = 0; i < model->outputs().size(); i++) {
                 proc.output(i).tensor().set_element_type(ov::element::f32);
             }
             model = proc.build();
-            ov::set_batch(model, batchSize);
+            if (FLAGS_bs) {
+                if (std::any_of(inputs.begin(), inputs.end(), [](const ov::Output<ov::Node>& i) {
+                        return (dynamic_cast<const ov::op::v0::Parameter&>(*i.get_node()).get_layout()).empty();
```
Comment (ilya-lavrenov) [inline samples/cpp/speech_sample/main.cpp:129]: we have `ov::layout::get_layout`

**Comment by mgrdv** [comment]
I close this PR because other PR https://github.com/openvinotoolkit/openvino/pull/10254 with adding layout as parameter was successfully merged

**Pull request closed without merging**